src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/utils/CustomPostgreSQL94Dialect.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.cft.utils;
```

Comment:
I would place this under the config package since it is referred from the application.yaml